Defs/Incidents/Incidents_TiberiumArrival.xml

```diff
@@ -6,7 +6,7 @@
 		<label>green tiberium asteroid</label>
 		<targetType>BaseMap</targetType>
 		<workerClass>TiberiumRim.IncidentWorker_GreenAsteroid</workerClass>
-		<baseChance>25.0</baseChance>
+		<baseChance>80.0</baseChance>

```

Comment:
These rate changes are _Ridiculous_. The base game rates for events are topped out at 5. Having a 50 times greater chance for a tiberium asteroid as opposed to say, an eclipse, is unacceptable.